introduction-to-metamask-api/index.md

```diff
@@ -0,0 +1,389 @@
+# Introduction to the MetaMask dApp API
+
+If you are planning to integrate authentication and identity into your Web3 applications, getting acclimated with the MetaMask API is a great way to start learning the capabilities afforded to you working with the MetaMask Extension and learning your way around the Web3 stack in general.
+
+## TOC
+
+- [What is the MetaMask API?](#what-is-the-metamask-api)  
+- [What is JSON RPC?](#what-is-json-rpc)  
+- [Essential Tools](#essential-tools)  
+  - [Ethereum Provider](#ethereum-provider)  
+  - [MetaMask RPC API](#metamask-rpc-api)
+  - [MetaMask API Playground](#metamask-api-playground)  
+  - [Chainlist](#chainlist)
+  - [Eserialize](#eserialize)
+- [Examples of Usage](#examples-of-usage)  
+- [Connecting to MetaMask Mobile with SDK](#connecting-to-metamask-mobile-with-sdk)
+- [Smart Contract Example](#smart-contract-example)  
+
+## What is the MetaMask API?
+
+Whenever we use a browser like Chrome, Firefox, or Brave and a wallet extension like MetaMask installed, we can use the Ethereum provider (as specified by EIP-1193) injected into the browser page at `window.ethereum`.
+
+We use this provider with our dapp to request users’ Ethereum accounts, read on-chain data and have the user sign messages and transactions.
+
+[Should we have a link to a docs page that describes what happens when multiple wallets are installed?]
+
+We can tell in the developer tools when MetaMask is installed and enabled through the developer tools, using chrome you can look into the Sources/Page, and you will see an icon that represents MetaMask.
+
+![](./images/01-sources-page.png)
+
+Page is used for viewing available resources on the current webpage. 
+Under the Top level, we see the representation of the main document and all of its resources.
+
+If we see MetaMask represented by a cloud icon here, we know that the MetaMask is installed and not disabled.
+
+The `window.ethereum` object is being injected into the page, allowing us to interact with MetaMask and Ethereum.
+
+We can directly access and call the `window.ethereum` object in our developer tools console. Let’s say that we wanted to get the current chainId…
+
+![](./images/02-console-eth-chainid.png)
+
+By running the code above in our console, and if we are currently connected to Polygon, we would see ‘0x89’ returned from this method call.
+
+As you get started with using MetaMask API methods, you might want to check out some of our [beginner-level videos](https://www.youtube.com/watch?v=03lbmYrawV8) from our Lead DX Engineer [Gui Bibeau](https://twitter.com/guibibeau) that can help get you started working with the MetaMask API.
+
+## What is JSON RPC?
```

Comment:
I would maybe introduce the relation ship between JSON RPC and MM before describing what it is. Maybe a single sentance in the API section above:

MetaMask API calls are defined with the JSON-RPC specs that we explain further down here [link to the explainer]